### DISCUSSION TITLE
is it possible to use IE instead of chorme?

### DISCUSSION THREAD

**Discussion by bigbattle45** [KevinBytesTheDust/CVAmp#196] [General]
course chrome eat to much ram

**Comment by jlplenio** [comment]
Hello @bigbattle45 and thank you for your question. Currently, there is no option to swap browsers. Generally, I could implement it, but this will break an unknown amount of functionality. Because every browser or rather, Twitch interacting with different browsers, handles communication/cookies/localstorage differently. Feature like status boxes, logged-in or chat for example. If I would switch in the current state, I am guessing only sending viewers will work, without anything else.
